File: blog-server/src/module/admin/login/model/Login.param.ts
import { IsString, IsEmail } from 'class-validator'


export class SignInParam {
    @IsString({ message: 'code必须的字符类型' })
    code: string
    
    @IsString({ message: 'password必须的字符类型' })
    password: string

    @IsString()
    captcha: string
}


export class ChankTokenParam {
    @IsString({ message: 'eMail必须的字符类型' })
    token: string
}

File: BlogServer/Blog.Model/Param/LoginParam.cs

namespace Blog.Model.Param
{
    public class RegisterLoginParam
    {
        public required string Name { get; set; }
        public required string Code { get; set; }
        public required string Password { get; set; }
    }

    public class SignInParam
    { 
        public required string Code { get; set; }
        public required string Password { get; set; }
        public required string Captcha { get; set; }
    }

    public class ChankTokenParam
    {
        public required string Token { get; set; }
    }
}
